grails-app/controllers/au/org/ala/volunteer/UserController.groovy

```diff
@@ -359,12 +361,21 @@ class UserController {
         sw.stop()
         log.debug("User.show()#taskList ${sw.toString()}")
 
+        // Find out if the user is a validator for any of the projects/institutions of the tasks in the list
+        taskList.viewList.each { Map row ->
+            def taskProject = Project.get(row.projectId as long)
```

Comment:
I believe `taskService.getNotebookTaskList` is only used in this controller action so these bits that hit the db should probably be moved inside the service method.